bundle/edu.gemini.pot/src/main/java/edu/gemini/spModel/gemini/flamingos2/Flamingos2.java

```diff
@@ -879,6 +900,10 @@ public static double getCustomMaskSetupSec() {
         return 30 * 60;
     }
 
+    public double getReacquisitionTime() { return 6 * 60; }
```

Comment:
I guess you can use `IMAGING_SETUP_TIME_PWFS2` here

Reply:
Not really, IMAGING_SETUP_TIME_PWFS2 and reacquisition time are different things (it's a coincidence that they are equal here). Reacquisition time is used only for spectroscopy.

Reply:
We can't make this `static` since it overrides a method in `ItcOverheadProvider`.
I marked it `@Override` instead.

Reply:
This makes me raise an eyebrow. We have:

`SPInstObsComp.reacquisitionTime(ISPObservation)` which returns 0 and is not overridden here.

Then we have `ItcOverheadProvider.reacquisitionTime()` which reutrns 6 minutes.

Is this correct? Should the two not be the same? This seems dubious.